java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -268,6 +268,36 @@ long getDefaultOutput() {
         }
     }
 
+    private static final class CollectAggregation extends Aggregation {
+        private final boolean includeNulls;
+
+        public CollectAggregation(boolean includeNulls) {
```

Comment:
This looks mostly good. What do you think about an enum class for `NullPolicy`?
```java
public enum NullPolicy
{
    EXCLUDE(false),
    INCLUDE(true);

    public NullPolicy(boolean includeNulls) { this.includeNulls = includeNulls; }

    final boolean includeNulls;
};
```
This will be analogous to the `null_policy` in `libcudf`, and is more readable than a bare boolean. It will be useful in the other APIs as well, but we can start its use here.

Reply:
Good suggestion, updated.